lms/djangoapps/certificates/queue.py

```diff
@@ -337,10 +337,10 @@ def add_cert(self, student, course_id, forced_grade=None, template_file=None, ge
                 str(exc)
             )
 
-        # Check if the student is whitelisted
-        if is_whitelisted:
+        # Check if the student is on the allow list
+        if is_allowlisted:
             LOGGER.info(
-                "Student %s is whitelisted in '%s'",
+                "Student %s is on the certificate allow list in '%s'",
```

Comment:
Nit/question: have we standardized on "allowlist" vs "allow list"?

Reply:
I don't think we have. I think we had silently agreed to "allowlist" and I went ahead and started messing things up :P. I will update this to be consistent.